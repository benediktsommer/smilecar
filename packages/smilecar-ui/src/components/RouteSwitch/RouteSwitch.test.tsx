import { render } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';

import { RouteSwitch } from './';

describe('Test <RouteSwitch />', () => {
  it('should render an html markup', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <RouteSwitch />
      </BrowserRouter>
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
