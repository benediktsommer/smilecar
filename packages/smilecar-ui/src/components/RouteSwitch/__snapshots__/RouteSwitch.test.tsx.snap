// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <RouteSwitch /> should render an html markup 1`] = `
<DocumentFragment>
  <div
    class="MuiBox-root MuiBox-root-1"
  />
</DocumentFragment>
`;
